content/platform/data-services-cloud-console/home.md

```diff
@@ -1,177 +1,42 @@
-
-### Authorization Policies
-
-The client can only receive properties from the authorized API resources based on the Role Base Access Control for the user who created the client-credential pair (client-id and client-secret). This authorization derives from the organization, capability and scope (roles) that the associated user is assigned. As the result, the authorization for the client application will inherit the user's permission who created the client-application registration under the API Gateway. Note that subsequent changes to the user's permission after the client application registered will impact the response returned based on current authority.
-
-### The API End Points (baseURL) for each DSCC Region
-
-The REST API for DSCC requires the client application to issue the REST API request to the URL that is associated with the DSCC instance deployed at the associated region of the storage array. As of November 2021, here are the  Domain URLs where client application must use as the base-URL to the resource path of REST API.
-
-| DSCC Region  | base-URL                       |
-| ------------ | ------------------------------ |
-| EU Central   | https://eu1.data.cloud.hpe.com |
-| AP Northeast | https://jp1.data.cloud.hpe.com |
-| US West      | https://us1.data.cloud.hpe.com |
-
-### Asynchronous Response
-
-All of the REST API operations are stateless in nature. One example is such as POST. In that scenario the task resource will return a response with HTTP code 202 "Accepted" and the reference to the task as follows:
-
-```md
-Response: 202 Accepted
-```
-
-```md
-{
-   "taskURi": "/api/v1/tasks/{task id}"
-}
-```
-
-In order to ensure the completion of this remote procedural call through POST, the users will use the task resource to query the status of this asynchronous task. 
-
-For more information about Data Services Cloud Console, please see this [website](https://www.hpe.com/us/en/storage/data-services-cloud-console.html?jumpid=ps_7niqt41kbx_aid-520042862&ef_id=EAIaIQobChMI9ZfRzLev9AIVx97ICh3b_QC0EAAYASAAEgKpxPD_BwE:G:s&s_kwcid=AL!13472!3!518496504236!e!!g!!data%20services%20cloud%20console!12802096712!120307197503&).
-
-```md
-/api/v1/tasks/{task id}
-
-GET responses
-{
-  state: {state ENUM}
-}
-
-state ENUM:
-- UNSPECIFIED
-- INITIALIZED
-- RUNNING
-- FAILED
-- SUCCEEDED
-- TIMEDOUT
-- PAUSED
-```
+<iframe width="560" height="315" src="https://www.youtube.com/embed/g3UO0S-4r6I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Suggest to add a heading for the title of the video:

#### Introduction to Data Services Cloud Console API
The embedded iframe syntax here.

Also I am wondering if it would not be better to move the embedded video before the list of blog posts.